zmsbackend/tests/Zmsbackend/Process/Api/ProcessFinishedTest.php

```
@@ -23,6 +26,14 @@ public function testRendering()
 
         $entity = (new \BO\Zmsbackend\Process\Service\Process)->readEntity($process->id, new \BO\Zmsbackend\Helper\NoAuth);
         $this->assertEquals('blocked', $entity->status);
+
+        $historyEntry = $this->readHistoryEntry((int) $process->id);
+
+        $this->assertIsArray($historyEntry);
+        $this->assertSame((int) $process->id, (int) $historyEntry['process_id']);
+        $this->assertSame(HistoryService::STATUS_COMPLETED, $historyEntry['status']);
+        $this->assertSame(\App::$now->format('Y-m-d H:i:s'), $historyEntry['finalized_at']);
+        $this->assertSame('08912345678', $historyEntry['telephone']);
```

PHPStan (2.2.7)

[error] Method BO\Zmsbackend\Tests\Process\Api\ProcessFinishedTest::testRendering() should return Psr\Http\Message\ResponseInterface but return statement is missing.


(return.missing)